TestModels/Aggregate/runtimes/python/pyproject.toml

```diff
@@ -0,0 +1,26 @@
+[tool.poetry]
+name = "simple-aggregate"
+version = "0.1.0"
+description = ""
+authors = ["AWS <aws-arg-dafny@amazon.com>"]
+packages = [
+    { include = "simple_aggregate", from = "src" }
+]
+# Include all of the following .gitignored files in package distributions,
+# even though it is not included in version control
+include = ["**/smithygenerated/**/*.py", "**/internaldafny/generated/*.py"]
+
+[tool.poetry.dependencies]
+python = "^3.11.0"
+DafnyRuntimePython = "^4.4.0"
+# TODO: Depend on PyPi once Smithy-Python publishes their Python package there
```

Comment:
nit: I suppose these comments aren't strictly accurate anymore? If I recall correctly, we plan to maintain a Smithy-Python fork indefinitely?

Reply:
Hopefully we don't maintain the fork indefinitely.

This comment is still accurate; if Smithy-Python were to publish their runtime to PyPI we should be able to depend on it immediately